src/Our.Umbraco.Ditto/ComponentModel/Attributes/AppSettingProcessorAttribute.cs

```diff
@@ -1,30 +1,27 @@
-﻿namespace Our.Umbraco.Ditto
-{
-    using System;
+﻿using System;
 
+namespace Our.Umbraco.Ditto
+{
     /// <summary>
     /// The AppSetting value attribute.
     /// Used for providing additional information about an AppSetting item to aid property value conversion.
     /// </summary>
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
-    public class AppSettingAttribute : DittoValueResolverAttribute
+    public class AppSettingProcessorAttribute : DittoProcessorAttribute
```

Comment:
I'm wondering if we drop the `AppSetting` attribute/resolver from Ditto core, (move it over to the labs repo)?

When it was first introduced, there was some discussion (in #72) about this.  My current thinking is that AppSettings are nothing to do with `IPublishedContent`.  By the same rationale, I'd have the same concerns about `UmbracoDictionary` attribute/resolver.

(Let's cut the fat, keep the Ditto core lean - move concept code over to the labs repo)

Any thoughts?